.circleci/config.yml

```diff
@@ -197,6 +197,10 @@ workflows:
           filters:
             tags:
               only: /^([0-9]+\.){3}dev[0-9]+/
+      - pr-preview: # Make a preview branch for each PR
+          filters:
+            tags:
+              only: /^([0-9]+\.){3}dev[0-9]+/
```

Comment:
Do we want to run this on nightly? I guess it doesn't hurt but I doubt anyone will do so?

Reply:
Hm, it's not configured to at the moment, IIUC. It shouldn't be too hard to refactor it to do so; I guess instead of "pr-2146" we'd configure it to be "nightly-0.69.3.dev20201019" or sth, and directly putting "streamlit==0.69.3.dev20201019" in the requirements.txt instead of uploading to the wheel to s3.

Anyways right now, navigating to the correct CircleCI page to get the nightly build link is a bit of a pain (unlike PR previews which show up in Github directly) Let's say we'll patch it in as soon as anyone asks for it!

Reply:
I believe this filter triggers it for nightly. That regex is the pattern that matches nightly. I think if we just remove it, it wont trigger during nightly but we can leave it in :) I'm not positive but I think the PR checks are not using these filters. Or I could be not understanding the filters and it won't trigger for nightly and it is required for PRs though then I don't understand how that would work since that regex wouldn't match any PR...